docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.8'
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8000:8000"
+    depends_on:
+      - redis
+
+  redis:
+    image: redis:alpine
+
+  nginx:
```

Comment:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.

Reply:
@Aiderbot specifically for deploying with Coolify, we don't actually need nginx and certbot. We just need Redis + the Python image.